context.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+
+
+class Context(ABC):
+    @abstractmethod
+    @property
```

Comment:
Please, change order of decorators:
property
abstractmethod
instead of:
abstractmethod
property

Reply:
Ok, ready